app/services/salesman/update.rb

```diff
@@ -0,0 +1,38 @@
+module Services
+  module Salesman
+    class Update
+      def initialize(id, name, status)
```

Comment:
Suggestion here is to receive a salesman object instead an `id` with your params to update.
```suggestion
      def initialize(salesman, update_params)
```